x-pack/plugins/fields_metadata/common/latest.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
Liked the idea to call it the latest, you can then switch the versions in this file and others could still continue to use the latest. Hopefully it does not break for them as they will not be aware that latest has now changed. What about also exposing this V1 in addition to latest. This way consumers has choice to either import latest or specific version

Reply:
It makes sense, also the [documentation](https://docs.elastic.dev/kibana-dev-docs/versioning-interfaces) suggests it! 

The reason I avoided this is that we don't only export types but also runtime types, which would increase the size of the bundle. We can add this as soon as we need it and keep the bundle small otherwise :)